tests/wen_new_standard.test.ts

```diff
@@ -0,0 +1,687 @@
+import * as anchor from "@coral-xyz/anchor";
+import { Program } from "@coral-xyz/anchor";
+import { expect } from "chai";
+import { faker } from "@faker-js/faker";
+import { WenNewStandard } from "../target/types/wen_new_standard";
+
+import {
+  Keypair,
+  AccountInfo,
+  PublicKey,
+  SystemProgram,
+  VersionedTransaction,
+  TransactionMessage,
+  LAMPORTS_PER_SOL,
+  SYSVAR_RENT_PUBKEY,
+} from "@solana/web3.js";
+import {
+  TOKEN_2022_PROGRAM_ID,
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+  getAssociatedTokenAddressSync,
+  getMint,
+  getAccount,
+  getMetadataPointerState,
+  Mint,
+  Account,
+  MetadataPointer,
+  getTokenMetadata,
+  createApproveCheckedInstruction,
+  createAssociatedTokenAccountInstruction,
+  createTransferCheckedInstruction,
+} from "@solana/spl-token";
+import {
+  TokenMetadata,
+  createUpdateFieldInstruction,
+} from "@solana/spl-token-metadata";
+import {
+  getApproveAccountPda,
+  getExtraMetasAccountPda,
+  wnsProgramId,
+} from "../clients/wns-sdk/src";
```

Comment:
Move helpers into this file temporary.

And wnsProgramId can be replaced by program.programId.

Reply:
Will move to generated sdk after.